web-common/src/features/dashboards/filters/dimension-filters/DimensionFilter.svelte

```diff
@@ -39,6 +44,8 @@
   export let timeEnd: string | undefined;
   export let timeControlsReady: boolean | undefined;
   export let smallChip = false;
+  export let whereFilter: V1Expression;
+  export let dimensionThresholdFilters: DimensionThresholdFilter[];
```

Comment:
Is this future proofing? It is always set to empty array in usages.

Please remove this if there is not intention of ever adding measure filters here, we can add it if and when we need it.

Reply:
yes, i'll remove it